lib/assets/SnapshotHelper.swift

```diff
@@ -10,15 +10,24 @@ import Foundation
 import XCTest
 
 var deviceLanguage = ""
+var alertDismissHandler: NSObjectProtocol?
 
 func setLanguage(app: XCUIApplication)
 {
     Snapshot.setLanguage(app)
 }
 
-func snapshot(name: String, waitForLoadingIndicator: Bool = true)
+func snapshot(name: String, waitForLoadingIndicator: Bool = true, waitForAlertsToBeHidden: Bool = true)
 {
-    Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator)
+    Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator, waitForAlertsToBeHidden: waitForAlertsToBeHidden)
```

Comment:
I like that the property uses the term 'dismiss' but then that verbiage is exchanged for 'hidden'. It seems like it might be more appropriate to keep the notion of 'dismissal' because that is really whats going on here. Definitely open to hearing your thoughts if you disagree!